main.go

```diff
@@ -21,6 +21,7 @@ import (
 	"github.com/decred/dcrd/dcrec/secp256k1"
 	"github.com/decred/dcrd/dcrutil"
 	"github.com/decred/dcrd/hdkeychain"
+	"github.com/decred/dcrwallet/wallet/udb"
```

Comment:
Please don't import this, it will eventually become an internal package

Reply:
What would be a better approach? Just copying the `CoinTypes` function over from the `udb` package? It is simple enough and I would rather not pull in all the dependencies from `udb`.

Reply:
That's fine.  We should also get these numbers in the chainparams.  The wallet code was written with the idea of eventually adding them to the params, and there's a test that the output remains the same even if the parameters change.

Reply:
I updated the PR with hardcoded coin type numbers. They should never be changed in the future again anyway, so this makes sure the code doesn't break if the chainparams are actually updated.